profile.html

```diff
@@ -8,13 +8,49 @@
   </head>
   <body>
     <header>
-      <h1>Quiz App</h1>
+      <h1>Quiz App - Profile</h1>
     </header>
-    <section class="max_dow">
-      <img
-        src="/pictures/Bildschirm­foto 2023-06-26 um 15.51.54.png"
-        alt="picture max dow" />
-    </section>
+    <main>
+      <section class="user--pic">
```

Comment:
in this section you have the image, headline, icons with numbers and about me...
i think user--pic is not the best name for that